Data-Analytics/Spark/k8s/DataProcessTransferFts-JarLocal-3.3.1.yaml

```diff
@@ -24,33 +22,35 @@ spec:
   mainApplicationFile: local:///tmp/DataProcessTransfer.jar
   mainClass: com.mapr.sparkdemo.DataProcessTransfer
   arguments:
-    - s3a://ezaf-demo/data/financial-partitioned      # source path
-    - parquet                                         # source format
-    - file:///mounts/data/financial-processed         # destination path
-    - parquet                                         # destination format
+    - s3a://ezua-demo/data/financial-partitioned       # source path
+    - parquet                                          # source format
+    - file:///mounts/shared-volume/financial-processed # destination path
+    - parquet                                          # destination format
   restartPolicy:
     type: Never
   imagePullSecrets:
-    - hpe-imagepull-secrets
+    - imagepull
   driver:
-    cores: 1
-    coreLimit: "1000m"
-    memory: "512m"
     labels:
       version: 3.3.1
     volumeMounts:
-      - mountPath: /mounts/data/
-        name: spark-data
+      - name: <username>-volume
+        mountPath: /mounts/<username>-volume
```

Comment:
You can leave the mount path static (do not include username there), as well as the volume name. Actually, only the PVC name matters.